resources/assets/uad_lists.json

```
@@ -14915,7 +14915,7 @@
   {
     "id": "tv.fubo.mobile.vpl",
     "list": "Misc",
-    "description": "fuboTV (https://play.google.com/store/apps/details?id=tv.fubo.mobile)\nLets you Watch live Sports, TV Shows, Movies & News\n",
+    "description": "fuboTV (https://play.google.com/store/apps/details?id=tv.fubo.mobile)\nLet's you Watch live Sports, TV Shows, Movies & News\n",
```

```suggestion
    "description": "fuboTV (https://play.google.com/store/apps/details?id=tv.fubo.mobile)\nLets you Watch live Sports, TV Shows, Movies & News\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.